lib/git-pr-release.js

```diff
@@ -36,7 +44,8 @@ module.exports = async function ({
   });
 
   let releasePr;
-  if (pullsRes.data.length === 0) {
+  const existingReleasePR = pullsRes.data.find((pr) => pr.title === RELEASE_TO_MASTER_PR_TITLE);
```

Comment:
If in the same repo there is another PR with the same title, that's someones fault. For release PRs created by automation, we can only have 1 at most.
Keep in mind this extra check could be unnecessary, think that we are querying PRs that go from release to main branch, and in theory we can not have more than one satisfying that criteria. I just added this extra rule because of the mysterious incidents that happened with a hotfix that was open and was being updated by the automation.